src/main/java/homework_3/pyramidprinter/PyramidPrinter.java

```diff
@@ -0,0 +1,28 @@
+package homework_3.pyramidprinter;
```

Comment:
The namings of packages fail my tests. Please fix

Reply:
How should I name the package?

Reply:
I dont understand with package name Homework_3. We must use the lowercase isn't?

Reply:
Fix it